.branch-variables.yml

```diff
@@ -10,7 +10,7 @@ distro_name: "fedora" # "fedora" or "rhel"
 distro_release: "rawhide" # "rawhide" or a number without quotation marks
 
 # The following only applies for rawhide.
-rawhide_fedora_version: 41 # number without quotation marks, or nothing if CI should not run for branched Fedora
+rawhide_fedora_version: 42 # number without quotation marks, or nothing if CI should not run for branched Fedora
 
 # List of all the branches which are currently supported and CI should be executed on these.
 supported_branches:
```

Comment:
You want to add `f41` to the list too. Development doesn't end there yet.

Reply:
Yeap thanks for catching this.